app/components/ProductList/productDetail.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './productdetail.css'
+import {Router, Route, Link, params, IndexRoute, hashHistory} from 'react-router'
+
+
+class ProductDetail extends React.Component {  
```

Comment:
Use functional component, because we don't have any state or lifecycle methods here